assets/konnectivity/konnectivity-server-services.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: konnectivity-server
+  labels:
+    app: kube-apiserver
+spec:
+  type: NodePort
+  selector:
+    app: kube-apiserver
+  ports:
+    - port: {{ .KonnectivityAgentPort }}
```

Comment:
Shouldn't this be the server port not the agent port?

Reply:
We have two services:
- konnectivity-server
  - agents on the control plane are connecting to the cluster service (using port)
  - agents on the data plane are connecting to the this service, but using NodePort
  - agentPort is opened on the server
- konnectivity-server-local
  - this is for the clients, clients are connecting to this port
  - server is opened also on the server

Naming may be a bit confusing, but this is what hypershift does right now.